# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/cookie/test/browser/browser.ini
# Commit: 001575dc6d78
# Full Hash: 001575dc6d78bf77bda6bd4949c887ba499e4fa7
# Author: Kris Maglione <maglione.k@gmail.com>
# Date: 2020-04-09 09:55:00
# Regressor Bug: 1582318
# File Overlap Count: 1
# Description:
#   Bug 1582318: Remove shutting-down processes from pool immediately. r=nika
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D69441
# ==============================================================================

diff -r 5349e5225018 -r 001575dc6d78 netwerk/cookie/test/browser/browser.ini
--- a/netwerk/cookie/test/browser/browser.ini	Wed Apr 08 21:10:10 2020 +0000
+++ b/netwerk/cookie/test/browser/browser.ini	Wed Apr 08 21:13:18 2020 +0000
@@ -13,7 +13,6 @@
 [browser_storage.js]
 [browser_serviceWorker.js]
 [browser_sharedWorker.js]
-skip-if = fission #Bug 1582318
 [browser_sameSiteConsole.js]
 support-files = sameSite.sjs
 [browser_oversize.js]
